app/src/main/java/com/android/unio/ui/user/UserClaimAssociation.kt

```diff
@@ -66,39 +72,31 @@ fun UserClaimAssociationScreen(
         ) {
           Column(
               modifier = Modifier.padding(16.dp),
-              verticalArrangement = Arrangement.spacedBy(6.dp),
+              verticalArrangement = Arrangement.spacedBy(12.dp),
               horizontalAlignment = Alignment.CenterHorizontally,
           ) {
             Text(
-                context.getString(R.string.user_claim_association_you_can_either),
+                context.getString(R.string.user_claim_association_you_can_do_the_following),
```

Comment:
If you don't leave a choice to the user in this page, maybe just remove this text as it gives no more information
It seemed weird when reading it in the screen